css/services.css

```diff
@@ -0,0 +1,109 @@
+.services-header-section {
+    margin: 2rem 0rem;
+}
+
+.services-header-img {
+    overflow: hidden;
+}
+
+.services-page-title {
+    font-size: 2rem;
+}
+
+.services-box-grouping {
+    display: flex;
+    justify-content: space-between;
+    align-content: space-evenly;
+}
+
+.services-box {   
+    padding: 1.5rem;
+    border: black solid .005rem;
+    background: lightgray;
+}
+
+.services-box:first-child {
+    margin: 2rem 2rem 2rem 0;
+}
+
+.services-box:last-child {
+    margin: 2rem 0 2rem 2rem;
+}
+
+.learn-more-button {
+    background: white;
+    border: black solid .1rem;
+    border-radius: .5rem;
+    margin-top: 2rem;
+    padding: 1rem 3rem;
+}
```

Comment:
using rem units for box model elements could change your site in unexpected ways,
its usually safer just to use rems on font size.